src/components/Common/Breadcrumb.vue

```diff
@@ -143,22 +143,19 @@
 export default {
   name: 'CommonBreadcrumb',
   methods: {
+    index() {
+      return this.$route.params.index
+        ? this.$store.direct.getters.index.getOneIndex(this.$route.params.index)
```

Comment:
Suggestion: un-bloat your code by using `mapGetters`

Reply:
Yes, I know mapGetter but finally, in use, I find it's simpler and faster to use the global store object (no import to add, syntax easier to remember).